### ISSUE TITLE
Set event’s target to the target of the last tuple in event’s path, that is either tuple or preceding tuple, whose target is non-null.

### ISSUE THREAD

**Issue by fanbinghuadev** [whatwg/dom#707]
https://dom.spec.whatwg.org/commit-snapshots/6e33c904f01c39f6d6a1565757da383089cde382/#dispatching-events

"Set event’s target to the target of the last tuple in event’s path, that is either tuple or preceding tuple, whose target is non-null"

Why event.target = the last tuple in event's path , is the first tuple in event's path?

**Comment by annevk** [comment]
Thanks for filing this, but I'm afraid I don't quite follow the question. Could you elaborate somehow?

**Comment by annevk** [comment]
The last that is/precedes tuple. So the higher up the tree we are, the higher the currently-exposed target is.

**Comment by fanbinghuadev** [comment]
But the event's path is filled from inner to outter，means the first tuple is event target，the second tuple is event target's parent...

**Comment by annevk** [comment]
Yes, in which case the tuple with the non-null target will always be or precede the tuple currently being iterated over, right?

**Comment by fanbinghuadev** [comment]
"The last" means the tuple with the non-null target?

**Comment by annevk** [comment]
The last of the tuple(s) with a non-null target that is also preceding or equal to _tuple_. (There can be multiple due to shadow trees.)

**Comment by fanbinghuadev** [comment]
Sorry,my english is bad! Thank you very much!I understand “the last”that it is in an array.

**Comment by annevk** [comment]
No worries, I'll leave this open for a bit to see if someone can come up with a better way of phrasing this. It's a tad confusing.
